Stripe/StripeiOSTests/PaymentSheet+APITest.swift

```diff
@@ -450,6 +449,101 @@ class PaymentSheetAPITest: XCTestCase {
         waitForExpectations(timeout: 10)
     }
 
+    /// Tests that when update is called while another update operation is in progress we ignore the in-flight update
+    func testUpdate_FirstUpdateIsIgnored() {
+        var intentConfig = PaymentSheet.IntentConfiguration(mode: .payment(amount: 1000, currency: "USD")) { _, _ in
+            // These tests don't confirm, so this is unused
+        }
+        let expectation = expectation(description: "Updates")
+        var flowController: PaymentSheet.FlowController!
```

Comment:
If we delete this it'll cause a crash. If we don't store the FlowController in a variable I noticed the `sut` instance being deallocated for some strange reason but storing in a explicit local var resolved the issue.

Reply:
Ah ok, it's deallocated because nothing holds onto `sut` until after the `create` callback returns.  You can either hold onto the flow controller in the `update` callbacks e.g.
```
                flowController.update(intentConfiguration: intentConfig) { [sut] _ in
                   XCTFail("This update call should be ignored in favor of the second update call")
                }
```
or leave this and add a comment explaining.

Reply:
Hmm I must be doing something wrong cause I see the crash when updating the test to:
And I also get a warning `Capture 'sut' was never used`
```swift
sut.update(intentConfiguration: intentConfig) { [sut] _ in
   XCTFail("This update call should be ignored in favor of the second update call")
}

intentConfig.mode = .setup(currency: nil, setupFutureUsage: .onSession)
sut.update(intentConfiguration: intentConfig) { error in
    XCTAssertNil(error)
    expectation.fulfill()
}
```

Reply:
You need to capture [sut] in the second `update` callback, too.

But re: the warning, ehhh yeah it's not worth doing this.  Let's just keep the approach you have now.